### PR TITLE
Updates Scanners Tab

### LINKED ISSUE #832: Add Akasha Scanner to scanner page
https://github.com/xenesty/AkashaScanner

### LINKED ISSUE #1054: Update scanner versions to 3.5 for Kamera/Artiscan/Adepti

### PR DESCRIPTION
**Pull request by PolomoPT**
Resolve #832
Resolve #1054

Messes with formatting a little, fixes issues with whitespace (is that the term?)

well, it's dandy. my commits are a little messy due to technical issues, but should be fine.

### CODE REVIEW

File: apps/frontend/src/app/PageScanner/index.tsx [hunk 1]
```diff
@@ -31,7 +32,16 @@ import AdScanner from './AdeptiScanner.png'
 import Amenoma from './Amenoma.png'
 import Artiscan from './artiscan.png'
 import GIScanner from './GIScanner.png'
-
+import AkashaScanner from './AkashaScanner.png'
+import SvgIcon from '@mui/material/SvgIcon'
+const Weapons = () => (
```
Comment (nguyentvan7) [inline apps/frontend/src/app/PageScanner/index.tsx:37]: Should be able to reuse `Assets.svg.anvil` the same way it is used in `Header.tsx`

File: libs/localization/assets/locales/en/artifact.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
     "circlet": "Circlet of Logos"
   },
   "info": {
-    "title": "Artifact Editing/Management Guide",
+    "title": "cucucucucucuu",
```
Comment (nguyentvan7) [inline libs/localization/assets/locales/en/artifact.json:48]: ?

File: apps/frontend/src/app/PageScanner/index.tsx (hunk 1 above)
Comment (PolomoPT) [reply apps/frontend/src/app/PageScanner/index.tsx:37]: i mean i tried but i couldn't. do you mean like `import Assets.svg.anvil from ____` and then `<SqBadge ____ > <Anvil />`?

Comment (nguyentvan7) [reply apps/frontend/src/app/PageScanner/index.tsx:37]: You'll have to `import Assets from '../Assets/Assets'` then do it as `<SqBadge>{Assets.svg.anvil}</SqBadge>`

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**